pwr2.c

```diff
@@ -0,0 +1,22 @@
+#include <stdio.h>
```

Comment:
Perfect job! 💯 the only suggestion I'd have is to have casted the results as ints so it "looks" nicer. Besides that, great job!